package.json

```diff
@@ -54,8 +54,8 @@
     "prettier": "^1.13.4",
     "pretty-bytes": "^4.0.2",
     "pretty-quick": "^1.6.0",
-    "react": "16.12.0",
-    "react-dom": "16.12.0",
+    "react": "16.12.0 || ^17.0.1",
+    "react-dom": "16.12.0 || ^17.0.1",
```

Comment:
Since they don't include 15 or 16 for devdependencies, is there any reason to match against 16?

Reply:
Who are "they" and where don't they include 15 or 16 to devdeps?

Reply:
"They" are the people who own this `package.json` and I mispoke, they didn't do `15 || 16`, so why now do `16 || 17` for devDependencies?

Reply:
It's a fail safe. It doesn't hurt to leave it there since 16 and 17 are compatible (which is not the case for 15 and 16)

Reply:
Compatible in what way? They're breaking versions, so at least some incompatibility must exist.

Either way I guess it's not a blocker.

Reply:
They are not breaking versions. 16 and 17 are 100% compatible with each other. That's the hole point of React 17. Please read the changelog before making bold statements.

Reply:
my dude:

> React 17 is a major release because these changes are potentially breaking

Reply:
My dude. How about you actually read the changelog?

https://es.reactjs.org/blog/2020/10/20/react-v17.html

Reply:
SCROLL DOWN.

Reply:
![Screen Shot 2020-11-26 at 9 08 00 AM](https://user-images.githubusercontent.com/334809/100377865-faa3c900-2fc6-11eb-92f4-8a29b923d857.png)

Reply:
Anyway, there's no sense in having 16 || 17 in devDependencies since yarn will install 17 anyway, see yarn.lock change

Reply:
You can see that [Rollup is set up to removed _react_ and _react-dom_](https://github.com/reach/router/blob/master/rollup.config.js#L16) from the bundle, thus the devDependancy really does not need to change.
```suggestion
    "react-dom": "16.12.0 || ^17.0.1",
```

Reply:
> hole point of React 17

I think you mean, "The whole point...". 😉